### PR TITLE
add apriori algorithm and correct test_cases y labels in desicison_tree.py

### PR DESCRIPTION
**Pull request by menghuu**
Hi, in `machine_learning\descision_tree.py`, the test_case's y_label is not correct. They should be `np.sin(test_cases)`

### CODE REVIEW

**Comment by mongoose64** [comment]
Hi, I am unfamiliar with github, so, the two different commits are in one pull-request.
did i need close this pr?

**Comment by cclauss** [comment]
I revert the change to decision_tree.py so you can make that on a _separate_ PR.

Please add at least one doctest to apriori.py so that we can merge this PR.

File: machine_learning/apriori.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+if __name__ == '__main__':
+    datas = [[1,3,4],[2,3,5],[1,2,3,5],[2,5]]
+    min_support = 0.5
+    min_conf = 0.5
+    lks, ck_and_supports = generate_lks(datas, min_support)
+    print("lks are: ", lks)
+    rules = generate_rules(lks, ck_and_supports, min_conf)
+    print("rules are: ", rules)
+    pass
```
Comment (onlinejudge95) [inline machine_learning/apriori.py]: Remove this line

**Comment by onlinejudge95** [comment]
Needs doctests for defined methods

**Comment by menghuu** [comment]
I don't have time to maintain it, so I close it.

**Pull request closed without merging**